static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -0,0 +1,464 @@
+import React from 'react'
+import { FaInfoCircle, FaPlus } from 'react-icons/fa'
+import PropTypes from 'prop-types'
+import ExternalLink from '../ExternalLink/ExternalLink'
+import EDSCIcon from '../EDSCIcon/EDSCIcon'
+import Button from '../Button/Button'
+import TourThumbnail from '../../../assets/images/tour-video-thumbnail.png'
+import './SearchTour.scss'
+
+export const TOTAL_STEPS = 12
+
+const TourButtons = ({ stepIndex, setStepIndex }) => (
+  <div className="tour-buttons">
+    <Button
+      type="button"
+      bootstrapVariant="secondary"
+      bootstrapSize="sm"
+      onClick={() => setStepIndex(stepIndex - 1)}
+    >
+      Previous
+    </Button>
+    <Button
+      type="button"
+      bootstrapVariant="primary"
+      bootstrapSize="sm"
+      onClick={() => setStepIndex(stepIndex + 1)}
+    >
+      Next
+    </Button>
+  </div>
+)
+
+TourButtons.propTypes = {
+  stepIndex: PropTypes.number.isRequired,
+  setStepIndex: PropTypes.func.isRequired
+}
+
+const StepCounter = ({ currentStep }) => (
+  <p className="step-counter-text">
+    {currentStep}
+    {' '}
+    OF
+    {' '}
+    {TOTAL_STEPS}
+  </p>
+)
+
+StepCounter.propTypes = {
+  currentStep: PropTypes.number.isRequired
+}
+
+const commonStyles = {
+  tooltip: {
+    width: '25rem'
+  },
+  tooltipContent: {
+    fontSize: '0.875rem',
+    textAlign: 'left',
+    lineHeight: '1.5'
+  }
+}
+
+const TourSteps = (stepIndex, setStepIndex, setRunTour) => [
+  {
+    target: '.search',
+    content: (
+      <div>
+        <h2 className="tour-heading">Welcome to Earthdata Search!</h2>
```

Comment:
If you look back at the mockup, this text was formatted differently. It should be smaller with some wider letterspacing